lms/envs/common.py

```diff
@@ -1992,6 +1992,7 @@
     [u"za", u"Zhuang"],
     [u"zu", u"Zulu"]
 )
+ALL_LANGUAGES_DICT = dict(ALL_LANGUAGES)
```

Comment:
To improve the performance of checking membership in the LanguageProficiency serializer's code validation step.... which I just removed, and forgot to remove this along the way :)